.github/workflows/evaluate.yml

```diff
@@ -28,10 +28,52 @@ on:
         description: 'CUDA_VISIBLE_DEVICES.'
         type: string
         default: '0,1,2,3,4,5,6,7'
+      dependency_pkgs:
+        required: true
+        description: 'Dependency packages, you can also set a specific version'
+        type: string
+        default: 'pynvml packaging protobuf transformers_stream_generator transformers==4.44.0'
 
 
 jobs:
+  linux-build:
+    if: ${{github.event_name == 'schedule' || (!cancelled() && !inputs.offline_mode)}}
```

Comment:
should we change ti to `if: ${{!cancelled() && (github.event_name == 'schedule' || !inputs.offline_mode)}}` as did in daily_ete_test.yml?

Reply:
> should we change ti to `if: ${{!cancelled() && (github.event_name == 'schedule' || !inputs.offline_mode)}}` as did in daily_ete_test.yml?

no need. currently evaluate is triggered manually.